.github/workflows/test-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: test-deployment
+
+permissions:
+  contents: read
+
+on: pull_request
+jobs:
+  docker-build:
+    permissions:
+      contents: read
+      packages: write
+    uses: scality/workflows/.github/workflows/docker-build.yaml@v1
+    with:
+      name: runner-manager
+      namespace: scality
+  test-deployment:
+    needs: docker-build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: helm/kind-action@v1
+      - name: Edit the runner-manager image
+        working-directory: tests
+        run: |
+          cat <<EOF > kustomization.yaml
```

Comment:
The previous method should have worked

Reply:
I prefer this way. I think it looks cleaner. In fact I think it should be a file in tests folder